web/source/tsconfig.embedded.json

```diff
@@ -7,9 +7,7 @@
 		"kmwbase.ts",
 		"keymanweb.ts",
 		"kmwosk.ts", 
-		"kmwembedded.ts", 
-		"kmwcallback.ts",
-		"kmwkeymaps.ts",
+		"kmwembedded.ts",
```

Comment:
I presume the other files are removed because they are referenced in kmwbase.ts? The dependency chain could just be a little buried?

Reply:
Exactly.  That reference automatically draws them in, so there's no need to explicitly build the file.  I forgot to remove 'em earlier.